c2corg_ui/templates/xreport/view.html

```diff
@@ -137,7 +137,8 @@ <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" d
 
   ## ASSOCIATIONS
   % if not version:
-    <div class="view-details-associations col-xs-12 associations">
+    <div class="view-details-associations col-xs-12 associations" ng-init="a = detailsCtrl.documentService.document.associations"
+         ng-show="a.waypoints.length || a.routes.length || a.articles.length || a.outings.length">
       <span class="lead">
 ##         <div ng-show="userCtrl.auth.isAuthenticated()" class="add-association">
```

Comment:
Yes. @dkocich said he removed these commented lines in #1108. If we are not going to merge #1108, it would be good to remove these lines here.